testgrid/config.yaml

```diff
@@ -3600,8 +3630,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu-p100-beta
   - name: kops-aws-1.11
     test_group_name: ci-kubernetes-e2e-kops-aws-beta
-  - name: kubelet-1.11
+  - name: kubelet-1.11-conformance
```

Comment:
this is not the conformance job?

Reply:
The name change is unnecessary. Removed :)